<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profile</class>
 <widget class="QWidget" name="Profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>905</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Overview</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f5f7fa;</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>30</number>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>40</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 28px; font-weight: bold; color: #2c3e50; background: transparent;</string>
     </property>
     <property name="text">
      <string>Profile Overview</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="statsLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QFrame" name="totalUsersFrame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white; border-radius: 12px; border-top: 4px solid #667eea;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <layout class="QVBoxLayout" name="totalUsersLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <property name="leftMargin">
         <number>24</number>
        </property>
        <property name="topMargin">
         <number>24</number>
        </property>
        <property name="rightMargin">
         <number>24</number>
        </property>
        <property name="bottomMargin">
         <number>24</number>
        </property>
        <item>
         <widget class="QLabel" name="totalUsersIcon">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #e8eaf6; border-radius: 10px; font-size: 24px;</string>
          </property>
          <property name="text">
           <string>üë•</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="totalUsersValue">
          <property name="styleSheet">
           <string notr="true">font-size: 28px; font-weight: bold; color: #2c3e50; background: transparent;</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="totalUsersLabel">
          <property name="styleSheet">
           <string notr="true">font-size: 14px; color: #7f8c8d; background: transparent;</string>
          </property>
          <property name="text">
           <string>Total Users</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="pendingRequestsFrame_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
border-radius: 12px;
border-top: 4px solid #f5576c;  /* same color as Edit Profile button */
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <layout class="QVBoxLayout" name="pendingRequestsLayout_2">
        <property name="spacing">
         <number>12</number>
        </property>
        <property name="leftMargin">
         <number>24</number>
        </property>
        <property name="topMargin">
         <number>24</number>
        </property>
        <property name="rightMargin">
         <number>24</number>
        </property>
        <property name="bottomMargin">
         <number>24</number>
        </property>
        <item>
         <widget class="QLabel" name="pendingRequestsIcon_2">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #fff3e0; border-radius: 10px; font-size: 24px;</string>
          </property>
          <property name="text">
           <string>üìî</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pendingRequestsValue_2">
          <property name="styleSheet">
           <string notr="true">font-size: 28px; font-weight: bold; color: #2c3e50; background: transparent;</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pendingRequestsLabel_2">
          <property name="styleSheet">
           <string notr="true">font-size: 14px; color: #7f8c8d; background: transparent;</string>
          </property>
          <property name="text">
           <string>Total Faculty</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="totalStudentsFrame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white; border-radius: 12px; border-top: 4px solid #f093fb;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <layout class="QVBoxLayout" name="totalStudentsLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <property name="leftMargin">
         <number>24</number>
        </property>
        <property name="topMargin">
         <number>24</number>
        </property>
        <property name="rightMargin">
         <number>24</number>
        </property>
        <property name="bottomMargin">
         <number>24</number>
        </property>
        <item>
         <widget class="QLabel" name="totalStudentsIcon">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #fce4ec; border-radius: 10px; font-size: 24px;</string>
          </property>
          <property name="text">
           <string>‚ö°</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="totalStudentsValue">
          <property name="styleSheet">
           <string notr="true">font-size: 28px; font-weight: bold; color: #2c3e50; background: transparent;</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="totalStudentsLabel">
          <property name="styleSheet">
           <string notr="true">font-size: 14px; color: #7f8c8d; background: transparent;</string>
          </property>
          <property name="text">
           <string>Total Students</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="pendingRequestsFrame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white; border-radius: 12px; border-top: 4px solid #ffa647;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <layout class="QVBoxLayout" name="pendingRequestsLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <property name="leftMargin">
         <number>24</number>
        </property>
        <property name="topMargin">
         <number>24</number>
        </property>
        <property name="rightMargin">
         <number>24</number>
        </property>
        <property name="bottomMargin">
         <number>24</number>
        </property>
        <item>
         <widget class="QLabel" name="pendingRequestsIcon">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #fff3e0; border-radius: 10px; font-size: 24px;</string>
          </property>
          <property name="text">
           <string>‚è≥</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pendingRequestsValue">
          <property name="styleSheet">
           <string notr="true">font-size: 28px; font-weight: bold; color: #2c3e50; background: transparent;</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pendingRequestsLabel">
          <property name="styleSheet">
           <string notr="true">font-size: 14px; color: #7f8c8d; background: transparent;</string>
          </property>
          <property name="text">
           <string>Pending Requests</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="adminInfoFrame">
     <property name="styleSheet">
      <string notr="true">background-color: white; border-radius: 12px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="adminInfoMainLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="adminInfoHeader">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent; border-bottom: 2px solid #f0f0f0;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <layout class="QHBoxLayout" name="adminInfoHeaderLayout">
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>30</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="adminInfoTitle">
           <property name="styleSheet">
            <string notr="true">font-size: 20px; font-weight: bold; color: #2c3e50; background: transparent;</string>
           </property>
           <property name="text">
            <string>Admin Information</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="headerSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="adminInfoBody" native="true">
        <property name="styleSheet">
         <string notr="true">background: transparent;</string>
        </property>
        <layout class="QVBoxLayout" name="adminInfoBodyLayout">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="topMargin">
          <number>30</number>
         </property>
         <property name="rightMargin">
          <number>30</number>
         </property>
         <property name="bottomMargin">
          <number>30</number>
         </property>
         <item>
          <layout class="QGridLayout" name="fieldsGrid">
           <property name="spacing">
            <number>20</number>
           </property>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditEmail">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>45</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    padding: 10px 15px;
    border: 2px solid #e0e0e0;
    border-radius: 8px;
    font-size: 14px;
    background-color: #f8f9fa;
    color: #222222;        /* slightly dark text */
}
/* Read-only state */
QLineEdit[readOnly=&quot;true&quot;] {
    background-color: #f0f0f0;       /* softer to indicate non-editable */
    color: #555555;                  /* slightly faded text */
}

</string>
             </property>
             <property name="text">
              <string>admin@university.edu</string>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="lineEditDepartment">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>45</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    padding: 10px 15px;
    border: 2px solid #e0e0e0;
    border-radius: 8px;
    font-size: 14px;
    background-color: #f8f9fa;
    color: #222222;        /* slightly dark text */
}
/* Read-only state */
QLineEdit[readOnly=&quot;true&quot;] {
    background-color: #f0f0f0;       /* softer to indicate non-editable */
    color: #555555;                  /* slightly faded text */
}

</string>
             </property>
             <property name="text">
              <string>Electrical and Computer Engineering</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="labelEmail">
             <property name="styleSheet">
              <string notr="true">font-size: 13px; color: #718096; font-weight: bold; background: transparent;</string>
             </property>
             <property name="text">
              <string>Email Address</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelDepatment">
             <property name="styleSheet">
              <string notr="true">font-size: 13px; color: #718096; font-weight: bold; background: transparent;</string>
             </property>
             <property name="text">
              <string>Department</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="lineEditName">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>45</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    padding: 10px 15px;
    border: 2px solid #e0e0e0;
    border-radius: 8px;
    font-size: 14px;
    background-color: #f8f9fa;
    color: #222222;        /* slightly dark text */
}
/* Read-only state */
QLineEdit[readOnly=&quot;true&quot;] {
    background-color: #f0f0f0;       /* softer to indicate non-editable */
    color: #555555;                  /* slightly faded text */
}

</string>
             </property>
             <property name="text">
              <string>admin_user</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelUsername">
             <property name="styleSheet">
              <string notr="true">font-size: 13px; color: #718096; font-weight: bold; background: transparent;</string>
             </property>
             <property name="text">
              <string>Username</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="buttonsLayout">
           <item>
            <spacer name="buttonsSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="buttonEditEmail">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>40</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #f5576c;
    color: white;
    border: none;
    border-radius: 8px;
    font-size: 14px;
    font-weight: bold;
    padding: 10px 20px;
}

QPushButton:hover {
    background-color: #e5475c;
}

QPushButton:pressed {
    background-color: #d93f53; /* slightly darker for click feedback */
}

QPushButton:disabled {
    background-color: #e0e0e0;
    color: #999999;
}
</string>
             </property>
             <property name="text">
              <string>Edit Email</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>120</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
